lib/pg_ha_migrations/safe_statements.rb

```diff
@@ -170,6 +175,87 @@ def safe_remove_concurrent_index(table, options={})
     unsafe_remove_index(table, **options.merge(:algorithm => :concurrently))
   end
 
+  # TODO: figure out what options we actually want to allow
+  #
+  # we definitely don't want algorithm and unique
```

Comment:
Another comment to start a PR discussion

Reply:
I addressed this in [f882261](https://github.com/braintree/pg_ha_migrations/pull/85/commits/f882261491c0a8a72490fdc44ea3d75daf958efa) which allows you to specify `unique`, `where`, and `comment`. 

A few notes:
- A comment will only be added to the parent index (seems redundant to add it to the child indexes)
- You can create a unique index if it includes the partition key column(s), but it's hard to validate this before executing the DDL, so using this option may result in a PG error that bubbles up like so:

  > PG::FeatureNotSupported:
  ERROR:  unique constraint on partitioned table must include all partitioning columns
  DETAIL:  UNIQUE constraint on table "foos3" lacks column "created_at" which is part of the partition key.